moveit_core/kinematic_constraints/test/test_constraints.cpp

```diff
@@ -652,6 +653,11 @@ TEST_F(LoadPlanningModelsPr2, OrientationConstraintsSimple)
   robot_state.setVariablePositions(jvals);
   robot_state.update();
   EXPECT_FALSE(oc.decide(robot_state).satisfied);
+
+  jvals["r_wrist_roll_joint"] = boost::math::constants::pi<double>();
```

Comment:
I would appreciate a comment here, to make the test more readable. Something like:

// 2pi wraps to zero degrees

Reply:
Done.